Source/Shared/Library/ObjectSerializer.swift

```diff
@@ -0,0 +1,42 @@
+import Foundation
+
+/// Convert json string, dictionary, data to Codable objects
+public class ObjectConverter {
```

Comment:
Not sure about the name, but don't have any suggestions now 😛

Reply:
The name is not good, I agree